recipes/Kili_fine_tuning_tutorial.ipynb

```diff
@@ -0,0 +1,2081 @@
+{
```

Comment:
<p>we usually assume that score close to 1 is a good score</p><p> </p><p>so here, I would use (800-671)/800 to get the score instead</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1259/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>Agreed. I've made this change</p>